Resources/Other-Sources/sv.lproj/Localizable.strings

```diff
@@ -0,0 +1,46 @@
+"DifferentTemperatureFormat" = "Det är %@ %@ än %@%@.";
+"PrecipitationChanceFormat" = "%@ chans för %@ %@.";
+"SameTemperatureFormat" = "Det är %@ %@ som %@%@.";
+
+"Hotter" = "hetare";
+"Warmer" = "varmare";
+"Cooler" = "svalare";
+"Colder" = "kallare";
+/* The word temperature is added at the end since the word is needed to make the sentence understandable in this context */
+"Same" = "samma temperatur";
```

Comment:
I believe that the english sentence "suffers" from the same problem.
Eg. "It's the same today as yesterday" does not state what is the same as yesterday.
The localization I provided would translate to "It's the same temperature today as yesterday" in Swedish. I could change it to match the english sentence if thats preferable? Or maybe the english sentence should be changed?